### ISSUE TITLE
Arakoon config path has changed

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/openvstorage-health-check#213]
**On all our CI test environments** we noticed the following exception:
```
[EXCEPTION] Error during execution of <alba_health_check.AlbaHealthCheck object at 0x7fd74ab24510>.run
Traceback (most recent call last):
  File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 35, in <module>
    class HealthCheckController(object):
  File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 419, in HealthCheckController
    HealthCheckController.run_method(*arguments)
  File "/opt/OpenvStorage/ovs/lib/healthcheck.py", line 403, in run_method
    getattr(cl, option['function'])(logger)
  File "/opt/OpenvStorage/ovs/extensions/healthcheck/alba/alba_health_check.py", line 675, in run
    AlbaHealthCheck.get_disk_safety(logger)
  File "/opt/OpenvStorage/ovs/extensions/healthcheck/alba/alba_health_check.py", line 537, in get_disk_safety
    namespaces = AlbaCLI.run(command="get-disk-safety", config=config)
  File "/opt/OpenvStorage/ovs/extensions/healthcheck/helpers/albacli.py", line 156, in run
    raise RuntimeError(output['error']['message'])
RuntimeError: Arakoon_exc.Exception(4, "Not_found")
```

After looking into why the exception occurs I noticed the following:
```
root@ovs-node01-1604:~# alba get-disk-safety --config=arakoon://config/ovs/arakoon/arakoon-mybackend02-abm/config?ini=%2Fopt%2FOpenvStorage%2Fconfig%2Farakoon_cacc.ini
2016-12-14 11:55:08 899347 +0100 - ovs-node01-1604 - 9033/0 - alba/cli - 0 - warning - Arakoon_exc.Exception(4, "Not_found")
root@ovs-node01-1604:~# ovs config get /ovs/arakoon/arakoon-mybackend02-abm/config
root@ovs-node01-1604:~# 
```

But this one is present! `ovs config get /ovs/arakoon/mybackend02-abm/config`

**Comment by kinvaris** [comment]
https://github.com/openvstorage/framework/issues/1270

**Comment by kinvaris** [comment]
Fixed in `openvstorage-health-check-3.1.2-rev.348.45e4ce5`

**Comment by JeffreyDevloo** [comment]
Fixed by https://github.com/openvstorage/openvstorage-health-check/pull/220
